0.6-dev/index.bs

```diff
@@ -0,0 +1,1787 @@
+to be correct / consistent for points that fall within those extents. It may not be correct for any point of
+appropriate dimensionality.
+
+<div class=example>
+
+<pre class=include-code>
+path: examples/transformations/bijection.json
+highlight: json
+</pre>
+
+the input and output of the `forward` and `inverse` transformations are understood to be:
+
+<pre class=include-code>
+path: examples/transformations/bijection_verbose.json
+highlight: json
+</pre>
+
+</div>
+
+
+"multiscales" metadata {#multiscale-md}
+---------------------------------------
+
+Metadata about an image can be found under the "multiscales" key in the group-level OME-Zarr Metadata. 
+Here, "image" refers to 2 to 5 dimensional data representing image or volumetric data with optional time or channel axes.
+It is stored in a multiple resolution representation.
+
+"multiscales" contains a list of dictionaries where each entry describes a multiscale image.
+
+Each "multiscales" dictionary MUST contain the field "coordinateSystems", whose value is an array containing valid coordinate
+system metadata (see [[#coord-sys-md]]). The first entry of this array is the "default" coordinate system, and should be
+used for viewing and processing unless a use case dictates otherwise. It will generally be a representation of the image
+in its native physical coordinate system.
+
+The following MUST hold for all coordinate systems.
+The length of "axes" must be between 2 and 5 and MUST be equal to the dimensionality of the Zarr arrays storing the image data (see "datasets:path").
+The "axes" MUST contain 2 or 3 entries of "type:space" and MAY contain one additional entry of "type:time" and MAY contain one
+additional entry of "type:channel" or a null / custom type.  The order of the entries MUST correspond to the order of dimensions
+of the Zarr arrays. In addition, the entries MUST be ordered by "type" where the "time" axis must come first (if present),
+followed by the  "channel" or custom axis (if present) and the axes of type "space".  If there are three spatial axes where two
+correspond to the image plane ("yx") and images are stacked along the other (anisotropic) axis ("z"), the spatial axes SHOULD be
+ordered as "zyx".
+
+Each "multiscales" dictionary MUST contain the field "datasets", which is a list of dictionaries describing the arrays storing
+the individual resolution levels.  Each dictionary in "datasets" MUST contain the field "path", whose value is a string
+containing the path to the Zarr array for this resolution relative to the current Zarr group. The "path"s MUST be ordered from
+largest (i.e. highest resolution) to smallest.  Every Zarr array referred to by a "path" MUST have the same number of dimensions
+and MUST NOT have more than 5 dimensions.  The number of dimensions and order MUST correspond to number and order of "axes".
+
+Each dictionary in "datasets" MUST contain the field "coordinateTransformation", whose value is a dictionary that defines a
```

Comment:
is the field called `coordinateTransformations` or `coordinateTransformation`? I think there are examples of both.